system/system_error_monitor/launch/system_error_monitor.launch.xml

```diff
@@ -1,13 +1,5 @@
 <launch>
   <arg name="config_file" default="$(find-pkg-share system_error_monitor)/config/system_error_monitor.param.yaml"/>
-  <arg name="ignore_missing_diagnostics" default="false"/>
-  <arg name="add_leaf_diagnostics" default="true"/>
-  <arg name="data_ready_timeout" default="30.0"/>
-  <arg name="data_heartbeat_timeout" default="1.0"/>
-  <arg name="diag_timeout_sec" default="1.0"/>
-  <arg name="hazard_recovery_timeout" default="5.0"/>
-  <arg name="use_emergency_hold" default="false"/>
-  <arg name="use_emergency_hold_in_manual_driving" default="false"/>
```

Comment:
I think we can keep this arguments as is for now. We can revisit this when we refactor launch file.